src/api/python/cvc5.pxi

```diff
@@ -1014,40 +1014,34 @@ cdef class Solver:
         Supports the following uses:
 
         - ``Op mkOp(Kind kind)``
-        - ``Op mkOp(Kind kind, Kind k)``
         - ``Op mkOp(Kind kind, const string& arg)``
-        - ``Op mkOp(Kind kind, uint32_t arg)``
-        - ``Op mkOp(Kind kind, uint32_t arg0, uint32_t arg1)``
-        - ``Op mkOp(Kind kind, [uint32_t arg0, ...])`` (used for the TupleProject kind)
+        - ``Op mkOp(Kind kind, uint32_t arg0, ...)``
```

Comment:
One might argue that it is "more pythonic" to only offer one of these. (Note that you can do `mkOp(kind, *l)` if you have a list `l`). But It's fine like that.

Reply:
Yes, I agree. I'm in favor of removing support for list.